util/mkdb.c

```diff
@@ -123,7 +123,7 @@ typedef struct oentry {
         char    *intypes;
         int     (*iopadr)(void *, void *p);
         int     (*kopadr)(void *, void *p);
-        int     (*aopadr)(void *, void *p);
+        int     (*dopadr)(void *, void *p);
```

Comment:
I'm not familiar with this utility, but this struct still has thread which is removed now. Would rather see init, perf, deinit here too.

Reply:
yep